### ISSUE TITLE
sysctl cookbook no longer carries apply recipe

### ISSUE THREAD

**Issue by pu239ppy** [bloomberg/chef-bach#1116]
Seems as if of version 1.x apply is no longer there
[sysctl-repo](https://github.com/sous-chefs/sysctl/tree/v1.0.1/recipes)
````
================================================================================
  Recipe Compile Error
  ================================================================================

  Chef::Exceptions::RecipeNotFound
  --------------------------------
  could not find recipe apply for cookbook sysctl

  Cookbook Trace:
  ---------------
    /var/chef/cache/cookbooks/bcpc-hadoop/recipes/configs.rb:79:in `from_file'

  Relevant File Content:
  ----------------------
  /var/chef/cache/cookbooks/bcpc-hadoop/recipes/configs.rb:

   72:    package hwx_pkg_str(pkg, node[:bcpc][:hadoop][:distribution][:release]) do
   73:      action :upgrade
   74:    end
   75:  end
   76:
   77:  # incrase max_map_count
   78:  node.default['sysctl']['params']['vm']['max_map_count'] = (node.memory.total.to_i) / 16
   79>> include_recipe 'sysctl::apply'
   80:

  Platform:
  ---------
  x86_64-linux
````

**Comment by pu239ppy** [comment]
For now will try another build with `depends 'sysctl', '= 0.9.0'` in `bcpc-hadoop::metadat.rb`

**Comment by vt0r** [comment]
Fixed in #1118
